docs/guide/mavic-2-pro.md

```diff
@@ -42,4 +42,5 @@ You will find the following sample in this folder: [WEBOTS\_HOME/projects/robots
 #### mavic\_2\_pro.wbt
 
 ![mavic_2_pro.wbt.png](images/robots/mavic-2-pro/mavic_2_pro.wbt.thumbnail.jpg) This demonstration shows a DJI Mavic 2 PRO flying over a rural area.
-After the drone has taken off, you can control it with the keyboard of your computer.
+The controller `mavic2pro.c` allows you to control the drone, after it has taken off, with the keyboard of your computer.
+The controller `mavicGotoTargets.py` shows how to move to specific world coordinates using its GPS, imu and gyroscope.
```

Comment:
controller should probably be prefixed `mavic2pro` as the other is, not just mavic. Also the name should use snake case to be consistent with the other python controllers. As for the name itself, perhaps: `mavic2pro_waypoints` or `mavic2pro_patrol` might be better?